input/pagecontent/reference.md

```diff
@@ -209,6 +209,132 @@ In cases with no explicit or logical restrictions on rule ordering, users MAY li
 
 It is possible for a user to specify contradictory rules, for example, two rules constraining the cardinality of an element to different values, or constraining an element to different data types. Implementations SHOULD detect such contradictions and issue appropriate warning or error messages.
 
+#### Indented Rules
+
+Indentation before a rule is used to set a context for the [path](#fsh-paths) on that rule. When one rule is indented below another, the full path of the indented rule is obtained by prepending the path from the rule above. For example:
+```
+* name 1..1
+  * family 1..1
+```
+is equivalent to:
+```
+* name 1..1
+* name.family 1..1
+```
+In this case, the first rule sets the "context" of `name` for any subsequent indented rules. Since the second rule is indented below the first, the `name` context is prepended to resolve the full path, `name.family`.
+
+Note that [path rules](#path-rules) can be used to set a path as context for subsequent rules, without applying any rules at that path. If in the above example, only `name.family`, but not `name`, was meant to be constrained, the rules could be specified as:
+```
+* name
+  * family 1..1
+```
+The following restrictions apply to indentation:
+* Regardless of indent, all rules must start on their own line.
+* Rules cannot be preceded by non-whitespace characters on a line.
+* Rules can only be indented in increments of two spaces.
+* A rule cannot be indented below a rule which has no path. For example, the following is an error:
+  
+  ```
+  * obeys inv-1
+    * family 1..1
+  ```
+* Similarly, a rule which has no path cannot be indented. For example, the following is also an error:
+  
+  ```
+  * name 1..1
+    * obeys inv-1
```

Comment:
Although the first example is valid, the second has to change, since a `obeys` rule can have a path.

Reply:
Fixed by using `insert` here: https://github.com/HL7/fhir-shorthand/pull/95/commits/655e2738e35dd5da92a7ad335fd600398f102bc8.